src/components/CandidateCategoryTile.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'cozy-ui/react/I18n'
+import AppIcon from 'cozy-ui/react/AppIcon'
+import IconGrid from 'cozy-ui/react/Labs/IconGrid'
+import AppLinker, { generateWebLink } from 'cozy-ui/react/AppLinker'
+import { withClient } from 'cozy-client'
+
+class CandidateCategoryTile extends React.Component {
```

Comment:
Since this component is just a render do we have some advantages to keep in as a Class and not a function? Since we're doing more and more hooks, I'm wondering if we should start a new habit

Reply:
The reason I used a class was to be abble to use context variables for domain and secure, but I'll change that and then we can use simple functions 👍